Fix all backup dates and date formats in logs format examples in code

meramsey opened the pull request:
this fixes the dates and backups to use domain vs username in all files so its uniform.

meramsey committed:
backupUtilities.py

Updated to have a more logical human-readable Date_time format. This will make reviewing backup files by name way easier to work with and more standardized. The current date only shows the day of week and month and year not the day of the month. Those doing daily backups are going to have a hard time telling which 1 of 4 Thursday the default would have been.

From:
time.strftime("%I-%M-%S-%a-%b-%Y")
06-50-03-Thu-Feb-2018
backup-example-06-50-03-Thu-Feb-2018

To:
time.strftime("%m.%d.%Y_%H-%M-%S")
02.15.2018_06-50-03
backup-example-02.15.2018_06-50-03

Also goes with my other proposed pull request for Update backupManager.py #126
https://github.com/usmannasir/cyberpanel/pull/126

Thanks

meramsey committed:
Update backupUtilities.py

updated the format to match the new domain vs username format

meramsey committed:
Update backupManager.py

fix backup to use domain vs username and more logical date formatting.

meramsey committed:
Fix all backup dates and date formats in logs format examples in code

spyngamerman commented:
This has all the commits from the below plus the other files where the dates and backup commands would have been mismatched
#129 opened 5 hours ago by spyngamerman 
#127 opened 3 days ago by spyngamerman 
#126 opened 3 days ago by spyngamerman

minhazulOO7 commented:
@spyngamerman thanks for mentioning! I submitted that PR way before but didn't got any notable response that's why didn't bother with it anymore. Recently I moved from CyberPanel to cPanel because of my time constraint. If I can I will be helping CyberPanel if it's willing to get supported, not like ignoring devs and not replying etc. like that.

meramsey committed:
Fix hardcoded cyberpanel version 1.8 on login page.

meramsey committed:
Update my.cnf

Set MariaDB to default to utf8mb4_unicode_ci collation and utf8mb4 which are the currently recommended defaults for lots of CMS and scripts. These support all the proper language sets across the board.
https://mariadb.com/kb/en/library/setting-character-sets-and-collations/#example-changing-the-default-character-set-to-utf-8

meramsey committed:
Update emailLimits.html

Fix typo: "Emai Limit" to "Email Limit"

meramsey committed:
Fix typo: "Emai Limit" to "Email Limit"

meramsey committed:
remaining old date formats

meramsey committed:
Merge pull request #1 from spyngamerman/patch-6

Patch 6

meramsey committed:
Merge pull request #2 from spyngamerman/patch-5

Update my.cnf

meramsey committed:
Merge pull request #3 from spyngamerman/patch-3

Update backupManager.py

meramsey committed:
Merge pull request #4 from spyngamerman/patch-2

Patch 2

meramsey committed:
Fix typo: Emai Limit to Email Limit

meramsey committed:
Fix remaining date formats

meramsey committed:
modified:   IncBackups/IncBackups.py
	modified:   backup/backupManager.py
	modified:   plogical/backupManager.py
	modified:   plogical/backupUtilities.py

meramsey committed:
Merge branch '1.8.0' into master

meramsey committed:
Merge pull request #5 from spyngamerman/master

Master

meramsey committed:
CSF integration improvements. Security home page show all security plugins. Standardize dates. Backups based on domain vs username.

meramsey committed:
Revert "CSF integration improvements. Security home page show all security plugins. Standardize dates. Backups based on domain vs username."

This reverts commit 2c04125eeced4eae61a99890246a41e0539ec6b7.
